src/azure-cli/azure/cli/command_modules/netappfiles/commands.py

```diff
@@ -114,25 +114,24 @@ def load_accounts_command_groups(self, netappfiles_accounts_sdk):
                          client_factory=accounts_mgmt_client_factory,
                          doc_string_source='azure.mgmt.netapp.models#NetAppAccount',
                          exception_handler=netappfiles_exception_handler)
-
         g.custom_command('remove', 'remove_active_directory',
                          client_factory=accounts_mgmt_client_factory,
                          doc_string_source='azure.mgmt.netapp.models#NetAppAccount',
                          exception_handler=netappfiles_exception_handler)
 
 
 def load_account_backup_command_groups(self, netappfiles_account_backups_sdk):
-    with self.command_group('netappfiles account backup', netappfiles_account_backups_sdk) as g:
+    with self.command_group('netappfiles account backup', netappfiles_account_backups_sdk, is_preview=True) as g:
```

Comment:
Why do we add `preview` here since SDK is stable version?

Reply:
@jsntcy this backup feature is only in private preview and is only properly set up in 2 testing regions. Also it has been unstable and still being worked on.